package.json

```
@@ -132,20 +132,30 @@
                         "when": "workspacePlatform == linux",
                         "command": "workbench.action.terminal.new"
                     },
+                    {
+                        "id": "python.createPythonFile",
+                        "title": "Create a Python file",
+                        "description": "Create a new file and save it as \".py\" to start with your Python development. \n[Create Python File](command:toSide:workbench.action.files.newUntitledFile)",
+                        "media": {
+                            "svg": "resources/walkthrough/open-folder.svg",
+                            "altText": "Open a a Python file or a folder with a Python project."
+                        },
+                        "when": ""
+                    },
                     {
                         "id": "python.selectInterpreter",
                         "title": "Select a Python Interpreter",
                         "description": "Choose which Python interpreter/environment you want to use for your Python project.\n[Select Python Interpreter](command:python.setInterpreter)\nTip: Reload the window if you installed Python but don't see it in the list (``Developer: Reload Window`` command in the [Command Palette](command:workbench.action.showCommands))",
                         "media": {
-                            "svg": "resources/walkthrough/python-interpreter.svg",
+                            "svg": "resources/walkthrough/python-interpreter-v2.svg",
                             "altText": "Selecting a python interpreter from the status bar"
                         },
                         "when": ""
                     },
                     {
                         "id": "python.runAndDebug",
                         "title": "Run and debug your Python file",
-                        "description": "[Open](command:toSide:workbench.action.files.openFile) or [create](command:toSide:welcome.showNewFileEntries) a Python file - make sure to save it as \".py\". \n Click on the play button on the top right of the editor, or press F5 when on the file and select \"Python File\" to run with the debugger. \n  \n[Learn more](https://code.visualstudio.com/docs/python/python-tutorial#_run-hello-world)",
+                        "description": "If you haven't already, [open](command:toSide:workbench.action.files.openFile) or [create](command:toSide:welcome.showNewFileEntries) a Python file - make sure to save it as \".py\". \n Click on the play button on the top right of the editor, or press F5 when on the file and select \"Python File\" to run with the debugger. \n  \n[Learn more](https://code.visualstudio.com/docs/python/python-tutorial#_run-hello-world)",
```

I don't think this change adds anything to the step. If you want to keep it, I would suggest the following:
```suggestion
                        "description": "If you haven't done so already, [open](command:toSide:workbench.action.files.openFile) or [create](command:toSide:welcome.showNewFileEntries) a Python file - make sure to save it as \".py\". \n Click on the play button on the top right of the editor, or press F5 when on the file and select \"Python File\" to run with the debugger. \n  \n[Learn more](https://code.visualstudio.com/docs/python/python-tutorial#_run-hello-world)",
```